packages/oruga/src/components/datepicker/DatepickerMonth.vue

```diff
@@ -0,0 +1,467 @@
+<template>
+    <section :class="monthClasses">
+        <div :class="monthBodyClasses">
+            <div class="datepicker-months">
```

Comment:
Why?

Reply:
I have tried to transpose this thing looking the oruga `DatepickerTable` and the buefy one

Reply:
I mean the third div static class

Reply:
I have to fix it

Reply:
done